src/test/java/com/example/employee/controller/EmployeeControllerTest.java

```diff
@@ -97,9 +97,14 @@ void getEmployeeById() throws Exception {
                 .andExpect(status().isOk())
                 .andReturn().getResponse().getContentAsString();
 
+        mockMvc.perform(MockMvcRequestBuilders.get("/employee/{id}", 1200L))
```

Comment:
Let's extract this case to its own test method and let's narrow down the exception to 404